articles/node-js-vs-django/index.md

```diff
@@ -0,0 +1,83 @@
+Node.js and Django are technologies used in building versatile web and mobile applications. Both have an excellent user interface, are open, and do not require a license to operate, and therefore they have a vast number of users. Node.js is a framework that JavaScript uses on a client-side web application. Django works on [Python applications](https://www.javatpoint.com/python-applications) to help web developers build apps more quickly.
```

Comment:
Django does not work on Python applications. It is used to build fully functional websites. Applications can be many. The link that you have given lists various applications possible with Python. Therefore, change the last sentence.